mac

```diff
@@ -128,34 +128,58 @@ else
   fancy_echo "Homebrew already installed. Skipping ..."
 fi
 
+brew_tap 'thoughtbot/formulae'
+
 fancy_echo "Updating Homebrew formulas ..."
 brew update
 
+# Dotfiles
+if ! command -v rcup >/dev/null; then
```

Comment:
It's either that `rcm` was added to Laptop before the `brew_install_or_upgrade` function was added or that there's something about `thoughtbot/formulae` that doesn't make them respond correctly to the `brew_is_installed` and `brew_is_upgradable` functions:

```
Installing parity ...
Warning: thoughtbot/formulae/parity-0.9.2.1 already installed

Installing rcm ...
Warning: thoughtbot/formulae/rcm-1.3.0 already installed
```